src/transactions/feed/TransactionFeedV2.test.tsx

```diff
@@ -236,17 +223,14 @@ describe('TransactionFeedV2', () => {
     expect(tree.getByText('transactionFeed.allTransactionsShown')).toBeTruthy()
   })
 
-  it('renders GetStarted if SHOW_GET_STARTED is enabled and transaction feed is empty', async () => {
+  it('renders GetStarted if transaction feed is empty', async () => {
     mockFetch.mockResponse(
       typedResponse({
         transactions: [],
         pageInfo: { hasNextPage: false, endCursor: '', hasPreviousPage: false, startCursor: '' },
       })
     )
     jest.mocked(getFeatureGate).mockImplementation((gate) => {
```

Comment:
these `false` mocks with a single gate seem unnecessary now given that a beforeEach resets them.